conf/defaults.ini

```diff
@@ -901,6 +901,10 @@ plugin_catalog_url = https://grafana.com/grafana/plugins/
 # tuning. 0 disables Live, -1 means unlimited connections.
 max_connections = 100
 
+# allowed_origins is a comma-separated list of origins that can establish connection with Grafana Live.
+# If not set then origin will be matched over root_url. Supports globbing: see https://github.com/gobwas/glob.
```

Comment:
I do not insist, but I wonder if this can be solved without an external dependency? gobwas/glob has nice features, but we kill most of them by splitting the string at commas (i.e. a valid gobwas' glob pattern `{foo.*,bar.*}` would probably be incorrectly split into `{foo.*` and `bar.*}`.

A naive globbing function matching `?` and `*` should be trivial to write, but from the memory I only have an IOCCC implementation in mind - https://www.ioccc.org/2001/schweikh.c - which is terse but unreadable. Rob Pike's approach to RegExps could also be converted to a glob matcher.

Also we have a tiny `convertGlobToRegex` function in the frontend, which can be ported to Go to use regexps instead.

Reply:
We kill only `{x,y}` syntax to set several origin patterns, replacing them with `x,y` which is more natural for Grafana config. What do you think if I update a description saying that we support wildcard symbol in patterns and remove link to gobwas/ws for now. I don't see a huge problem continue using gobwas/ws internally to match individual patterns.

Reply:
Updating the comment could be sufficient for this issue.